_pages/topics/git_recovery.md

```diff
@@ -21,6 +21,9 @@ Rosszul írtad le a commitodat, vagy esetleg rossz változtatások kerültek bel
 ### Megoldás
 A `git commit --amend` lecseréli a `HEAD` commitot, így ahelyett hogy lenne egy "add cool feature" commit és egyből utána egy "fix embarrassing typo in previous change" commit, csak egy "add cool feature" commit van typo nélkül.  A parancs amúgy ugyanúgy működik mint mindig, tehát a `git add` vagy `git commit --patch` hívása ugyanúgy szükséges a változtatások hozzáadásához, de ha csak a commit leírását akarod változtatni akkor nem.
 
+### Git merge
+A jelenlegi és egy adott másik branch szétválása óta történt változásokat fésüli össze arra a branchre amin jelenleg vagyunk. Többekközött a git pull is ezt használja hogy a remote repositoryba feltöltött változásokat beleolvassza a local repository kódjába, de bármelyik két tetszőleges branch változásait összefésülhetjük. Különbség a rebase-héz képest hogy a merge nem írja át/fésüli össze a két branch commit historyát az összegzés során, csak létrehoz egy merge commitot amiben az összes változás egyszerre jelenik meg.
```

Comment:
This is backwards.  Merge crates a merge commit, which has multiple ancestor commits.  The merge commit itself does not contain all the changes.  For an example, look at fe72a15ea65621b4fa0183b4d514022e1aa43c23 in this repo, with `git show`.

A merge commit is IMHO more accurately thought of as recording a resolution of two diverging lines of history.

Also, this is kind of a nitpick, but pull doesn't act on two repositories, it acts on two branches, one of which is usually a remote tracking branch.